### PR TITLE
analyzed some non-functional aspects

### PR DESCRIPTION
**Pull request by jolomarci**

### CODE REVIEW

**Review by vassbence — APPROVED** [review]
LGTM

File: doc/non-functional-tests-doc.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Non-functional tests
+
+## Web.dev measurements
+
+With the help of https://web.dev/measure/ we measured how the application performs in different aspects. The site gives us scores for the Performance, Accessibility Best Practices and SEO aspects.
+
+<figure><img src="web.dev_results.png"><figcaption>The results</figcaption></figure>
+
+The page produced desirable results, the only field where the we should consider some changes was the Accessiblity.
+The application prohibits zooming on the pages, wich could be problematic for users with low vision. Even though the website was made for university students, we should make it accessible for everyone.
```
Comment (vassbence) [inline doc/non-functional-tests-doc.md:10]: which

File: doc/non-functional-tests-doc.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+<figure><img src="web.dev_accessibility.png"><figcaption>Accessiblity notes</figcaption></figure>
+
+Another point, where we could improve the application is the choice of colours. Witc better contrast ratio, the site would be better to look at for longer periods of time.
```
Comment (vassbence) [inline doc/non-functional-tests-doc.md:14]: With

**Review by vassbence — CHANGES_REQUESTED** [review]
Nice work, mentioned some typos

File: doc/non-functional-tests-doc.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# Non-functional tests
+
+## Web.dev measurements
+
+With the help of https://web.dev/measure/ we measured how the application performs in different aspects. The site gives us scores for Performance, Accessibility, Best Practices and SEO aspects.
+
+<figure><img src="web.dev_results.png"><figcaption>The results</figcaption></figure>
+
+The page produced desirable results, the only field where the we should consider some changes was the Accessiblity.
```
Comment (Cuslika) [inline doc/non-functional-tests-doc.md:9]: -the

File: doc/non-functional-tests-doc.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+Another point, where we could improve the application is the choice of colours. With better contrast ratio, the site would be better to look at for longer periods of time.
+
+## Webpagetest.org results
+
+We also checked the site with another web tool. The results were the following:
+
+<figure><img src="webpagetest.org_results.png"><figcaption>The results</figcaption></figure>
+
+Here the results were also promosing with the exception of Security. The used Javascript libraries were safe, however there were some missing HTTP security headers. The page listed some examples, to choose from.
```
Comment (Cuslika) [inline doc/non-functional-tests-doc.md:22]: promosing typo

**Review by Cuslika — CHANGES_REQUESTED** [review]

**Review by Cuslika — APPROVED** [review]
Okay

**Review by vassbence — APPROVED** [review]
LGTM

**Pull request merged**